user_docs/en/userGuide.t2t

```diff
@@ -1998,15 +1998,11 @@ By default this command will cycle between the following modes:
 -
 
 There are more advanced sound split modes available in NVDA setting combo box.
-If you wish to adjust volume of all applications except for NVDA, consider using [the dedicated commands #OtherAppVolume].
 Please note, that sound split doesn't work as a mixer.
 For example, if an application is playing a stereo sound track while sound split is set to "NVDA on the left and applications on the right", then you will only hear the right channel of the sound track, while the left channel of the sound track will be muted.
 
 This option is not available if you have started NVDA with [WASAPI disabled for audio output #WASAPI] in Advanced Settings.
 
-Please note, that if NVDA crashes, then it won't be able to restore application sounds volume, and those applications might still output sound only in one channel after NVDA crash.
-In order to mitigate this, please restart NVDA.

```

Comment:
@mltony - should this section also be kept - i.e. is the issue surrounding sound split being stuck after a crash still an issue?

Reply:
Yes, this section should be kept. There is nothing we can do to restore volumes of apps when NVDA crashes.